datumaro/components/annotation.py

```diff
@@ -207,6 +208,15 @@ class Label(Annotation):
     label: int = field(converter=int)
 
 
+@attrs(slots=True, order=False)
+class HashKey(Annotation):
+    _type = AnnotationType.hash_key
+    hash_key: List[str] = field(factory=list, validator=default_if_none(list))
+    label: Optional[int] = field(
+        converter=attr.converters.optional(int), default=None, kw_only=True
+    )
```

Comment:
Why do we need `label` in `HashKey`?

Reply:
I think No. But in [`_check_annotations`](https://github.com/openvinotoolkit/datumaro/blob/3aa474226f19cee00f2e8e78347d8a11a4c4ff8a/datumaro/components/launcher.py#L71-L72) function checked whether each annotation included label or not. I think I had no choice but to add `label` for `HashKey`.

Reply:
I removed `label` for `HashKey` and updated checking label part of annotation in `_check_annotation`.